chart/values.yaml

```
@@ -530,3 +527,121 @@ mysql:
   extraVolumeMounts: |
     - mountPath: /docker-entrypoint-initdb.d
       name: init-scripts
+
+promtail:
+  enabled: true
+  fullnameOverride: promtail
+
+  initContainer:
+    enabled: true
+    image:
+      registry: docker.io
+      repository: busybox
+      tag: 1.33
+      pullPolicy: IfNotPresent
+    fsInotifyMaxUserInstances: 128
+
+  rbac:
+    pspEnabled: true
+
+  config:
+    serverPort: 3101
+    lokiAddress: http://loki:3100/loki/api/v1/push
+    snippets:
+      scrapeConfigs: |
+        - job_name: workspaces
+          pipeline_stages:
+            {{- toYaml .Values.config.snippets.pipelineStages | nindent 4 }}
+          kubernetes_sd_configs:
+            - role: pod
+          relabel_configs:
+            # Drop everything that is not a gitpod workspace
+            - source_labels:
+              - __meta_kubernetes_pod_label_app
+              - __meta_kubernetes_pod_label_component
+              regex: workspace
```

I think that the way it is now wouldn't work, it is trying to match the labels `app` and `component` and their values combined should be `workspace`.


We would need to choose between one of the two 👇 

```suggestion
              - __meta_kubernetes_pod_label_app
              - __meta_kubernetes_pod_label_component
              regex: gitpod;workspace
```
or

```suggestion
              - __meta_kubernetes_pod_label_component
              regex: gitpod;workspace
```